Lib/sysconfig/__init__.py

```diff
@@ -718,14 +718,17 @@ def expand_makefile_vars(s, vars):
     """
     import re
 
+    find_var1_re = re.compile(r"\$\(([A-Za-z][A-Za-z0-9_]*)\)")
+    find_var2_re = re.compile(r"\${([A-Za-z][A-Za-z0-9_]*)}")
```

Comment:
It seems like this code is not tested, so I would prefer to restore removed code rather than writing new code:

```py
_findvar1_rx = r"\$\(([A-Za-z][A-Za-z0-9_]*)\)"
_findvar2_rx = r"\${([A-Za-z][A-Za-z0-9_]*)}"
```

These variables should be copied from `Lib/sysconfig/__main__.py`.